### PR TITLE
SCO: Update UI based on UX team feedback #19965

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

Fix spacing issues in "Policies and Procedures" and "Averaging processing times" sections

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19965

Address 508 concerns in 
https://github.com/department-of-veterans-affairs/va.gov-team/issues/3580

## Testing done 
\[Provide link to QA Test Plan ticket (issue-template coming soon) for the relevant product/feature.
List the new/update unit-test(s) & e2e-test\(s) that have been successfully run before open this PR.
Also, if UI-testable (i.e., testable manually on Staging by QA, either via the browser or REST API-client), provide link to QA test-request ticket \[issue-template coming soon].]

See [VSA QA Process](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/qa/vsa-qa-process.md) for more info on Engineering/QA collaboration/engagement.

## Screenshots
Screen Shot 2019-11-19 at 1 20 11 PM
Screen Shot 2019-11-19 at 12 38 02 PM

## Acceptance criteria
- [ ] 

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the Pre-Launch Checklist
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] A link has been provided to the VSA-QA Test Plan ticket \[use the VSA-QA Test Plan Issue Template to create the ticket within va.gov-team repo].
- [ ] A link has been provided to the VSA-QA Test-Request ticket \[issue-template coming soon].
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/site/components/merger-crosslinks.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
   {% for group in crosslinks %}
 
     {% if group.heading != empty %}
-      <h3 class="va-nav-linkslist-heading">{{ group.heading }}</h3>
+      <h3 class="va-nav-linkslist-heading" tabindex="-1">{{ group.heading }}</h3>
```
Comment (cvalarida) [inline src/site/components/merger-crosslinks.html]: @1Copenut Does this look okay to you?

**Review by cvalarida — COMMENTED** [review]
Only thing I might have a concern on is the `tabindex`es. Pinging Trevor to make sure that passes his sniff check.

File: src/site/components/merger-crosslinks.html (hunk 1 above)
Comment (zurbergram) [reply src/site/components/merger-crosslinks.html]: added per his request in https://github.com/department-of-veterans-affairs/va.gov-team/issues/3580

Comment (cvalarida) [reply src/site/components/merger-crosslinks.html]: Awesome. Thanks for the cross-reference!

**Review by cvalarida — APPROVED** [review]

**Pull request merged**